### DISCUSSION TITLE
How can i provide custom value in TextInput ?

### DISCUSSION THREAD

**Discussion by goigarg** [filamentphp/filament#3397] [Help] [answered]
I want to provide a custom value on Textinput in All Pages

Something Like this 

`Forms\Components\TextInput::make('user_email')->value("example@example.com");`

**Comment by Z3d0X** [comment, marked as answer]
You mean a default value? `->default("example@example.com")`,
Or do you want https://filamentphp.com/docs/2.x/forms/getting-started#filling-forms-with-data
